<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <group ns="$(arg uav_name)">

    <node pkg="nodelet" type="nodelet" name="control_test" args="standalone mrs_testing/ControlTest" output="screen">

      <rosparam file="$(find mrs_testing)/config/control_test.yaml" />

      <!-- Subscribers --> 
      <remap from="~odometry_in" to="odometry/new_odom" />
      <remap from="~position_command_in" to="control_manager/position_cmd" />

      <!-- Publisher --> 
      <remap from="~goto_out" to="control_manager/goto" />
      <remap from="~goto_relative_out" to="control_manager/goto_relative" />
      <remap from="~goto_altitude_out" to="control_manager/goto_altitude" />
      <remap from="~set_yaw_out" to="control_manager/set_yaw" />
      <remap from="~set_yaw_relative" to="control_manager/set_yaw_relative" />

      <remap from="~set_trajectory_out" to="control_manager/mpc_tracker/set_trajectory" />

      <remap from="~switch_tracker_out" to="control_manager/switch_tracker" />

    </node>

  </group>

</launch>
